libs/zgpu/src/zgpu.zig

```diff
@@ -1716,3 +1716,9 @@ pub const bglBuffer = gpu.BindGroupLayout.Entry.buffer;
 pub const bglTexture = gpu.BindGroupLayout.Entry.texture;
 pub const bglSampler = gpu.BindGroupLayout.Entry.sampler;
 pub const bglStorageTexture = gpu.BindGroupLayout.Entry.storageTexture;
+
+test "use zpool package from zgpu?" {
+    const zpool = @import("zpool");
```

Comment:
Just added this test as a quick way to see whether the `zpool` package was accessible here.

```sh
$ zig test ./libs/zgpu/src/zgpu.zig 
./libs/zgpu/src/zgpu.zig:1721:19: error: unable to find 'zpool'
    const zpool = @import("zpool");
```

Reply:
`zig test` doesn't invoke the calls to `addPackage`, you can use the `--pkg-begin` and `--pkg-end` options to specify packages (see `zig test --help`)

Reply:
`zig test --pkg-begin zpool ./libs/zpool/src/zpool.zig --pkg-end ./libs/zgpu/src/zgpu.zig` should do it